moma_demos/piloting_demo/launch/sim.launch

```diff
@@ -5,8 +5,10 @@
     <arg name="use_fixed_realsense" default="false"/>
     <arg name="tool" default="panda_hand"/>
     <arg name="launch_navigation" default="true"/>
-    <arg name="world_name" default="$(find piloting_demo)/worlds/schutz.world"/>
+    <arg name="world_name" default="$(find piloting_demo)/worlds/jfloor.world"/>
     <arg name="use_global_map" default="true"/>
+    <arg name="global_map_config_file"   default="$(find piloting_demo)/maps/jfloor/map.yaml"/>
+    <arg name="global_map_cloud_file"     default="$(find piloting_demo)/maps/jfloor/map.pcd"/>
```

Comment:
Maybe we should have a `setup` param such that changing this single var we can easily swap setup and stick to the naming convention
```
<arg name="setup" default="jfloor"/>

<arg name="world_name"                default="$(find piloting_demo)/worlds/$(arg setup).world"/>
<arg name="global_map_config_file"    default="$(find piloting_demo)/maps/$(arg setup)/map.yaml"/>
<arg name="global_map_cloud_file"     default="$(find piloting_demo)/maps/$(arg setup)/map.pcd"/>
```